### PR TITLE
Connecting frontend to backend

### PR DESCRIPTION
**Pull request by scarletnguyen13**

### CODE REVIEW

File: client/src/components/SentimentAnalysis/Helpers.js [hunk 1]
```diff
@@ -8,12 +8,14 @@ const AnalysisInput = props => {
```
Comment (rob-marcus) [inline client/src/components/SentimentAnalysis/Helpers.js:8]: Please add a JSDoc somewhere to this with background on what this is doing/why/general context.

File: client/src/components/SentimentAnalysis/SentimentAnalysis.js [hunk 2]
```diff
@@ -1,32 +1,53 @@
...
 
-  const {
-    mockContentTerms,
-    mockHeaderTerms,
-    mockHeaderScore,
-    mockHeaderMagnitude,
-    mockContentScore,
-    mockContentMagnitude,
-  } = MOCK_DATA;
+  const submitForm = e => {
```
Comment (rob-marcus) [inline client/src/components/SentimentAnalysis/SentimentAnalysis.js:27]: `e` is a bad name.

File: client/src/components/SentimentAnalysis/Helpers.js (hunk 1 above)
Comment (kirakira0) [reply client/src/components/SentimentAnalysis/Helpers.js:8]: +1

Comment (scarletnguyen13) [reply client/src/components/SentimentAnalysis/Helpers.js:8]: It's just a textarea so I thought it's already self-explainatory :-?

Comment (rob-marcus) [reply client/src/components/SentimentAnalysis/Helpers.js:8]: I mean in reference to the entire doc. The title `helpers` doesn't really tell us what's in the code, and it's good style to just have a JSdoc giving an overview without having to dissect the code/what is being returned.

**Pull request merged**